ext/hash/tests/hash_update_stream_basic_001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Ensure hash_update_stream() always returns the same hash when $length = 0
+--CREDITS--
+Symeon Charalabides <symeon@systasis.com> - @phpdublin
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
+    die('SKIP - php version too low.');
+}
+if (!extension_loaded('hash')) die('skip hash extension not available');
+if (!extension_loaded('openssl')) die('skip openssl extension not available');
+require_once(dirname(__FILE__) . '/skip_mhash.inc'); ?>
```

Comment:
skip_mhash.inc is removed as of PHP 7.4.0, so trying to include it fails, which causes the test to be marked as borked. Actually, there is no need to check for the mash extension, since it is not used by the test at all.

Reply:
This is an old pull request. Fixed now.